### PR TITLE
[Docs] [Master] Adding multi-tenancy properties to config catalog

### PR DESCRIPTION
**Pull request by Chamindu36**
## Purpose
Adding the following configurations to the config catalog in the docs.
```
[multi_tenancy.stratos]
disable_tenant_management_emails = true
email_validation_mandatory_for_login = false
tenant_activation_moderated = false
super_admin_email = "super-admin-email"
skip_summary_generator = false
notification_email = "notification-email"
finance_notification_email = "finance-notification-email"
charge_on_registration = false
public_cloud_setup = true
google_analytics_url = "http://google-analytics.js"
sso_loading_message = "Loading..."
    [multi_tenancy.stratos.paypal]
    url = "Paypal URL"
    username = "username"
    password = "password"
    signature = "signature"
    environment = "live"
    [multi_tenancy.stratos.manager_service]
    url = "https://cloud-test.wso2.com/services/"
    admin_username = "admin"
    admin_password = "admin"

[multi_tenancy.usage_agent.data_persistence_task]
startup_delay_in_ms = "60000"
records_per_execution = "100"
execution_interval_in_ms = "-1"

[[multi_tenancy.throttling_manager_tasks]]
data_providers = [
    "org.wso2.carbon.throttling.manager.dataproviders.BillingDataProvider",
    "org.wso2.carbon.throttling.manager.dataproviders.UsageDataProvider"
]
    [multi_tenancy.throttling_manager_tasks.parameters]
    interval = "10"
    "delay" = "60"

[[multi_tenancy.cloud_services]]
name = "WSO2 API Manager"
default = true
key = "AM"
label = "API Manager"
link = "https://am.cloud.wso2.com"
description = "API Manager in the cloud"
[multi_tenancy.billing.package.demo]
name = "Demo"
subscription_charge = 0
    [multi_tenancy.billing.package.demo.users]
    limit= "1"
    charge= "0"
    [multi_tenancy.billing.package.demo.resource_volume]
    limit= "20"
    overuse_charge= "0"
    [multi_tenancy.billing.package.demo.bandwidth]
    limit= "50"
    overuse_charge= "0"

[multi_tenancy.billing.package.smb]
name = "SMB"
subscription_charge = "100"
    [multi_tenancy.billing.package.smb.users]
    limit= "unlimited"
    charge= "0"
    [multi_tenancy.billing.package.smb.resource_volume]
    limit= "50"
    overuse_charge= "0"
    [multi_tenancy.billing.package.smb.bandwidth]
    limit= "150"
    overuse_charge= "0"

[[multi_tenancy.billing.package.custom]]
name = "Professional"
subscription_charge = "500"
    [[multi_tenancy.billing.package.custom.users]]
    limit = "unlimited"
    charge = "0"
    [[multi_tenancy.billing.package.custom.resource_volume]]
    limit = "500"
    overuse_charge = "0"
    [[multi_tenancy.billing.package.custom.bandwidth]]
    limit = "1536"
    overuse_charge = "0.03"
```

## Goals
Fixes https://github.com/wso2/docs-apim/issues/1276 for the master branch in docs.

### CODE REVIEW

File: en/docs/reference/config-catalog.md [hunk 1]
```diff
@@ -4443,3 +4445,1562 @@ enable_email_domain = true</code></pre>
...
+url = "https://cloud-test.wso2.com/services/"
+admin_username = "admin"
+admin_password = "admin"
+</code></pre>
+                    </div>
+                </div>
+                <div class="doc-wrapper">
+                    <div class="mb-config">
+                        <div class="config-wrap">
+                            <code>[multi_tenancy.billing.package.demo]</code>
```
Comment (Mariangela) [inline en/docs/reference/config-catalog.md:4804]: The sample does not match this heading. Should this be `[multi_tenancy.stratos.manager_service]`
Screenshot 2020-10-21 at 21 51 51

File: en/docs/reference/config-catalog.md [hunk 2]
```diff
@@ -4443,3 +4445,1562 @@ enable_email_domain = true</code></pre>
...
+    <section>
+        <div class="mb-config-options">
+            <div class="superfences-tabs">
+
+                <input name="33" type="checkbox" id="_tab_33">
+                <label class="tab-selector" for="_tab_33"><i class="icon fa fa-code"></i></label>
+                <div class="superfences-content">
+                    <div class="mb-config-example">
+                        <pre><code class="toml">[multi_tenancy.stratos.manager_service]
+url = "https://cloud-test.wso2.com/services/"
```
Comment (Mariangela) [inline en/docs/reference/config-catalog.md:4795]: Double check whether this URL is correct.

File: en/docs/reference/config-catalog.md [hunk 3]
```diff
@@ -4443,3 +4445,1562 @@ enable_email_domain = true</code></pre>
...
+<div class="mb-config-catalog">
+    <section>
+        <div class="mb-config-options">
+            <div class="superfences-tabs">
+
+                <input name="46" type="checkbox" id="_tab_46">
+                <label class="tab-selector" for="_tab_46"><i class="icon fa fa-code"></i></label>
+                <div class="superfences-content">
+                    <div class="mb-config-example">
+                        <pre><code class="toml">[[multi_tenancy.billing.package.custom]]
```
Comment (Mariangela) [inline en/docs/reference/config-catalog.md:5747]: Isn't it incorrect to add double `[[ ]]` here?

Screenshot 2020-10-21 at 20 56 39

**Review by Mariangela — CHANGES_REQUESTED** [review]
@Chamindu36 , please note the review comments that I have added. Please make sure to pull my changes, which are in your Fork, to your local clone when addressing the review comments so that my changes do not get deleted.

File: en/docs/reference/config-catalog.md (hunk 2 above)
Comment (Chamindu36) [reply en/docs/reference/config-catalog.md]: This is an example URL. But I have updated this URL with a meaningful URL.

File: en/docs/reference/config-catalog.md (hunk 1 above)
Comment (Chamindu36) [reply en/docs/reference/config-catalog.md]: Fixed

File: en/docs/reference/config-catalog.md (hunk 3 above)
Comment (Chamindu36) [reply en/docs/reference/config-catalog.md]: Yes, it is incorrect. I have corrected that.

**Review by Mariangela — APPROVED** [review]

**Pull request merged**